lib/rails/generators/alchemy/install/templates/elements.yml.tt

```diff
@@ -1,6 +1,6 @@
-# == In this configuration you setup Alchemy´s element layouts.
+# == In this configuration, you set up Alchemy's element layouts.
 #
-# For further informations please see http://guides.alchemy-cms.com/create_elements.html
+# For further information please see http://guides.alchemy-cms.com/edge/elements.html
```

Comment:
Sure thing.  `grep`'ed for guides.alchemycms.com/edge/ and replaced with /stable/, so some other files got picked up.